backend/src/main/java/org/sefglobal/core/controller/MultiverseController.java

```diff
@@ -62,8 +67,11 @@ public University updateUniversity(@PathVariable long id,
     //todo: Implement an endpoint to delete university (change active status)
 
     @GetMapping("/ambassadors")
-    public Iterable<Ambassador> listAmbassador() {
-        return ambassadorRepository.findAll();
+    @ResponseStatus(value = HttpStatus.OK)
+    public Page<Ambassador> getAmbassadors(@RequestParam(defaultValue = "0") int limit,
+             @RequestParam(defaultValue = "10") int pageNumber) {
```

Comment:
Remove these `defaultValue` and add `@Range` annotation to validate the params

Reply:
Ayya it doesn't seem to work that's why I used `defaultValues`

Reply:
@jayasanka-sack We talked about this.

Reply:
according to this code default value for limit is 0 and pageNumber's default value is 10.

Anyway, @anjula-sack you have to validate the params.
Validation is more important than default values

Reply:
Can I just use `If` to validate instead of range

Reply:
if you use an if condition, you have to write a new exception class and throw that exception.

I think we can achieve this by using simple annotations.

Reply:
Ok I'll try again